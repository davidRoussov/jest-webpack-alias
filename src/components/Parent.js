import React, { Component } from 'react';
import Child from '@/components/Child';

class Parent extends Component {
  render() {
    return (
      <div>
        <Child/>
      </div>
    );
  }
}

export default Parent;
