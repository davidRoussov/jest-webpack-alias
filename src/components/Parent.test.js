import React from 'react';
import Parent from './Parent';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
      <Parent/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
