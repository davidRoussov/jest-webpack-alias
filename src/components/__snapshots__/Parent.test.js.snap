// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <h2>
      Hello, World!
    </h2>
  </div>
</div>
`;
